api/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[tool.poetry]
```

Comment:
Potentially just a quick explanation on the difference between this `pyproject.toml` and the `build.toml` file for clarity?

Reply:
build.toml is used when building the Docker image (see Dockerfile line 24)

- the main difference is that the `asf-hp-installer-chatbot` package is able to be linked via relative paths (line 37 of pyproject.toml)
- when building the image for linux/amd64 architectures, to avoid downloading and compiling CUDA dependencies that will result in 5x the image size, the link to the binaries are included directly (line 33, and 34 of build.toml)
- also when building the image, the source dependencies from `asf-hp-installer-chatbot` are directly copied, therefore included separately.